### DISCUSSION TITLE
Using dunder as attribute names in models?

### DISCUSSION THREAD

**Discussion by Kayvinh** [sqlalchemy/sqlalchemy#9823] [Usage Questions]
I had __ts_vector__ as one of my class attributes for my model and it didn't work.

Does SQLAlchemy not support using dunder as class atrributes for models anymore? The example I was following was from https://amitosh.medium.com/full-text-search-fts-with-postgresql-and-sqlalchemy-edc436330a0c

Just wondering what has changed? I tried searching it in the documentation but couldn't find it.

**Comment by zzzeek** [comment]
there's no limitation with single underscores for attributes, I would not use a double under like `__some_attr__`, that wouldn't be recognized, a leading double under like `__some_attr` I'd probably also avoid as it modifies Python's attribute behaviors.

but `ts_vector`?  no problem there, the database in question might have a problem with that as a column name though since it might be a reserved word
